.github/system_tests/pytest/test_memory_leaks.py

```diff
@@ -23,6 +25,7 @@ def run_finished_ok(*args, **kwargs):
     assert node.is_finished_ok, (node.exit_status, node.exit_message)
 
 
+@pytest.mark.usefixtures('aiida_profile')
```

Comment:
I've removed them, but I am not sure that it will work. These tests are called in a different way, outside of the normal test suite. They don't have the `conftest.py` which imports the `aiida.manage.tests.pytest_fixtures` which is responsible for loading the `aiida_profile` fixture.

Reply:
They did run, but that is only because we explicitly pass the `conftest.py` of the `tests` folder. I think these tests were placed outside of the suite originally when we were still half running `unittest` in the migration to `pytest`. I will move them to the `tests` folder now, since they don't need to be separated any longer.